pkg/scheduler/framework/plugins/podtopologyspread/common.go

```diff
@@ -33,17 +35,35 @@ type topologyPair struct {
 // and where the selector is parsed.
 // Fields are exported for comparison during testing.
 type topologySpreadConstraint struct {
-	MaxSkew     int32
-	TopologyKey string
-	Selector    labels.Selector
-	MinDomains  int32
+	MaxSkew            int32
+	TopologyKey        string
+	Selector           labels.Selector
+	MinDomains         int32
+	NodeAffinityPolicy v1.NodeInclusionPolicy
+	NodeTaintsPolicy   v1.NodeInclusionPolicy
+}
+
+func (tsc *topologySpreadConstraint) matchHonorNodeInclusionPolicies(pod *v1.Pod, node *v1.Node, require nodeaffinity.RequiredNodeAffinity) bool {
```

Comment:
nit: IMO matchNodeInclusionPolicies reads better, as our logic respects all policies - both Honor and Ignore effects.

Reply:
updated, thanks wei.